usr/src/uts/common/sys/fs/zfs.h

```diff
@@ -956,11 +956,18 @@ typedef enum {
  *		ZFS_EV_POOL_GUID	DATA_TYPE_UINT64
  *		ZFS_EV_VDEV_PATH	DATA_TYPE_STRING	(optional)
  *		ZFS_EV_VDEV_GUID	DATA_TYPE_UINT64
+ *
+ *	ESC_ZFS_HISTORY_EVENT
+ *
+ *		ZFS_EV_POOL_NAME	DATA_TYPE_STRING
+ *		ZFS_EV_POOL_GUID	DATA_TYPE_UINT64
+ *		ZFS_EV_HISTORY		DATA_TYPE_NVLIST
```

Comment:
It might be worth documenting what's in the nvlist, or at least pointing folks to `log_internal()` / `spa_history_log_sync()` which generates the nvlist.